docs/reference-architecture/video-transcoding/diagrams/figure3.svg

```diff
@@ -0,0 +1,6173 @@
+    <linearGradient
+       id="linear-gradient"
```

Comment:
This is where the gradient is defined... change the ID to something unique like `id="svg3--linear-gradient"`